boards/funhouse/definition.json

```diff
@@ -111,7 +111,29 @@
              "displayName":"Speaker/Piezo",
              "dataType":"bool",
              "hasPWM":true
-          }
+          },
+          {
+            "name":"D10",
+            "displayName":"D10",
+            "dataType":"bool"
+         },
+         {
+            "name":"D11",
+            "displayName":"D11",
+            "dataType":"bool"
+         },
+         {
+            "name":"D43",
+            "displayName":"D44 (UART RX)",
+            "dataType":"bool",
+            "uartRx": true
+         },
+         {
+            "name":"D43",
+            "displayName":"D43 (UART TX)",
+            "dataType":"bool",
+            "uartTx": true
+         }
```

Comment:
Now you've got two pins name "D43" 😅 

I need to augment our JSON validator so we can add more complex checks like these.

Reply:
Sorry, was working quickly. The first one is supposed to be D44 for its name. Put back.